wiki/FAQ/en.md

```diff
@@ -1,731 +1,99 @@
+### I am having an error "Bad Authentication Token". What should I do?
 
-If miss the timing to tap the starting circle point, miss the slider ticks, or miss the reverse arrow points, combo will break (it should trigger a combo break sound when the combo is high enough) but it is not considered a miss.
-However, if miss the ending circle point, combo does not break and does not increment by one.
-It is also not considered a miss; instead, a 100 (or katu for end of the combo) will be given for the entire slider, assuming at least the slider starting circle point done properly, and completed all the slider ticks/reverse arrow points when given.
-Notably, [Sudden Death][Sudden Death wikilink] game mod will not trigger its effect of immediate game over when missing the ending circle point (since the combo did not break, the Sudden Death effect was not triggered).
+### Can I use another username?
 
-These situations, caused by a slider, are collectively called **sliderbreak**.
-A max combo play can be known by checking the result screen for the _Perfect_ notice on the performance graph; not having the _Perfect_ notice, especially without any miss, usually means there is one or more sliderbreak occurred during play.
+### What is this voice status that I have? I also see some people having it.
 
-[Answer source with test example in gaming section of stackexchange](https://gaming.stackexchange.com/questions/221663/i-fcd-a-song-why-is-my-combo-lower-than-people-on-the-high-scores)
+## Wiki
 
-### osu!taiko
-#### What does "Use Taiko skin for Taiko mode" in Options under Skin section do?
-The button only works if a folder named `taiko` (capitalisation does not matter) inside the `Skins` folder of the `osu!` file directory existed.
+### Markdown supports HTML tags, can I use them?
 
-When enabled, it will use the `taiko` folder's skin elements when playing in osu!taiko, regardless of current skin's taiko elements.
-If the `taiko` folder was not found, or disabled, it will use the current skin's taiko elements when available; default if all/certain taiko skin element was not found.
+### Why does the file I uploaded not load on the osu!wiki?
 
-**Ancient trivia note:** `taiko` folder used to hold _[Taiko by LuiginHann][Taiko by LuigiHann link]_ skin, which could be downloaded from the now-deprecated `osume.exe` (an _osu!_ updater when _osu!_ need to update) under Skin tab.
-As the button does not check whether the folder contains elements from _LuigiHann's skin_, it was quickly noted by the community and used as a personal global osu!taiko skin folder.
+### How do I align the images?
 
-### Beatmapping
-#### "This beatmap you're trying to submit isn't owned by you!"
-_For unranked beatmaps only._
+### How would I know the changes looks like in the official page?
 
-![Example of an .osu file content][Beatmap_not_owned image]
+### Is there any difference between osu!wiki and GitHub Preview?
```

Comment:
Might as well make it plural: `### Are there any differences between osu!wiki and GitHub Preview?`